tests/foreman/virtwho/cli/test_esx_sca.py

```diff
@@ -502,3 +508,87 @@ def test_positive_remove_env_option(
+
+        :customerscenario: true
+
+        :CaseImportance: Medium
+
+        :BZ: 2173870
+        """
+
+        # create 3 hypersiors and each have 3 guests json file
+        json_file = Path("/tmp/fake.json")
+        data = hypervisor_fake_json_create(hypervisors=1, guests=1)
+        json_file.write_text(json.dumps(data))
+        # create 10 fake files in module_sca_manifest_org
+        virtwho_config_cli = target_sat.cli.VirtWhoConfig.create(form_data_cli)[
+            'general-information'
+        ]
+        command = get_configure_command(virtwho_config_cli['id'], module_sca_manifest_org.name)
+        deploy_configure_by_command(
+            command, form_data_cli['hypervisor-type'], org=module_sca_manifest_org.label
+        )
+        config_file_1 = get_configure_file(virtwho_config_cli['id'])
+        owner_1 = get_configure_option('owner', config_file_1)
+        rhsm_hostname_1 = get_configure_option('rhsm_hostname', config_file_1)
+        rhsm_username_1 = get_configure_option('rhsm_username', config_file_1)
+        rhsm_encrypted_password_1 = get_configure_option('rhsm_encrypted_password', config_file_1)
+        # create another org and create fake conf files in this org
+        ORG_DATA = {'name': f'virtwho_fake_{gen_string("alpha")}'}
+        org = target_sat.api.Organization(name=ORG_DATA['name']).create()
+        target_sat.api.Location(organization=[org]).create()
+        form_data_cli['organization-id'] = org.id
+        virtwho_config_cli = target_sat.cli.VirtWhoConfig.create(form_data_cli)[
+            'general-information'
+        ]
+        command = get_configure_command(virtwho_config_cli['id'], org.name)
+        deploy_configure_by_command(
+            command, form_data_cli['hypervisor-type'], debug=True, org=org.label
+        )
+        config_file_2 = get_configure_file(virtwho_config_cli['id'])
+        owner_2 = get_configure_option('owner', config_file_2)
+        rhsm_hostname_2 = get_configure_option('rhsm_hostname', config_file_2)
+        rhsm_username_2 = get_configure_option('rhsm_username', config_file_2)
+        rhsm_encrypted_password_2 = get_configure_option('rhsm_encrypted_password', config_file_2)
+        for i in range(5):
+            fake_conf_file = f"/etc/virt-who.d/virt-who-config-fake{i}.conf"
+            vw_fake_conf_create(
+                owner_1,
+                rhsm_hostname_1,
+                rhsm_username_1,
+                rhsm_encrypted_password_1,
+                fake_conf_file,
+                json_file,
+            )
+        for i in range(5, 10):
+            fake_conf_file = f"/etc/virt-who.d/virt-who-config-fake{i}.conf"
+            vw_fake_conf_create(
+                owner_2,
+                rhsm_hostname_2,
+                rhsm_username_2,
+                rhsm_encrypted_password_2,
+                fake_conf_file,
+                json_file,
+            )
+        vw_run_option("od")
```

Comment:
Explain `od` in comment please?

Reply:
> Explain `od` in comment please?

@lhellebr the od is the option for virt-who cli, 
`  -d, --debug           Enable debugging output
  -o, --one-shot        Send the list of guest IDs and exit immediately
`

Reply:
Have already add the comments